hw/top_darjeeling/data/top_darjeeling.hjson

```diff
@@ -1014,6 +1014,20 @@
       },
       attr: "ipgen"
     },
+    { name: "ac_range_check",
```

Comment:
@Razer6 ac_range_check is not actually found by the tools as the dependency is missing in the top_darjeeling.core file. I've tried to add it but there is an issue with the actual core file of ac_range_check.

Reply:
Fixed that.

Reply:
Thanks for fixing this @Razer6 . I see `ac_range_check.sv` has quite some syntax errors. Can you please take a look?